test/source/tests/page-recipe/settings-page-recipe.ts

```diff
@@ -14,6 +14,7 @@ import { KeyUtil } from '../../core/crypto/key';
 export type SavePassphraseChecks = {
   isSavePassphraseHidden?: boolean | undefined,
   isSavePassphraseChecked?: boolean | undefined
+  isInvalidKey?: boolean | undefined
```

Comment:
This check isn't related to "save passphrase", so the type name `SavePassphraseChecks` would no longer be relevant.
Moreover, these checks are supposed to be optional (can be skipped), which is not the case with invalid key -- we HAVE to dismiss the alert to proceed with the recipe, so conceptually this is not the best place for this parameter.
Can we put it somewhere to `ManualEnterOpts` maybe?